tests/test_attrs.py

```diff
@@ -0,0 +1,197 @@
+"""
+This file tests attrs support.
+
+See https://github.com/snok/flake8-type-checking/issues/77
+for discussion on the implementation.
+"""
+
+import textwrap
+
+import pytest
+
+from flake8_type_checking.codes import TC002
+from tests import _get_error
+
+
+@pytest.mark.parametrize(
+    'expected',
+    (
+        {'2:0 ' + TC002.format(module='decimal.Decimal')},
+        {'2:0 ' + TC002.format(module='decimal.Decimal')},
+    ),
+)
+def test_non_attrs_model(expected):
+    example = textwrap.dedent(
+        '''
+        from decimal import Decimal
+
+        class X:
+            x: Decimal
+        '''
+    )
+    assert _get_error(example, error_code_filter='TC001,TC002') == expected
```

Comment:
Are we not just running the same test twice here?

```suggestion
def test_non_attrs_model(expected):
    example = textwrap.dedent(
        '''
        from decimal import Decimal

        class X:
            x: Decimal
        '''
    )
    assert _get_error(example, error_code_filter='TC001,TC002') == {'2:0 ' + TC002.format(module='decimal.Decimal')}
```

Reply:
True, it was for my internal testing that I don't break functionality unrelated to `attrs`.